Herebyfile.mjs

```diff
@@ -1208,6 +1208,6 @@ export const installExtension = task({
         console.log(pc.yellowBright("\nExtension installed. ") + "To enable this extension, set:\n");
         console.log(pc.whiteBright(`    "typescript.experimental.useTsgo": true\n`));

```

Comment:
Is this going to be the final setting name?

Reply:
It's hardcoded in https://github.com/microsoft/vscode/blob/35d9d31a8c96ba3e545a1db67a91dc9876551cc9/extensions/typescript-language-features/src/extension.ts#L37 so we can't really change it immediately

Reply:
@mjbvz can we at least change how this looks on the UI so `Tsgo` is not awkwardly Pascal-cased?

![Uploading image.png…]()

Reply:
Mentioned here that extensions may not be able to change it: https://github.com/microsoft/vscode/issues/73374

@rzhao271  Can let the TS team know if there's a way around the auto casing in the settings UI

Reply:
I see the Tsgo setting in the [package.json](https://github.com/microsoft/vscode/blob/35d9d31a8c96ba3e545a1db67a91dc9876551cc9/extensions/typescript-language-features/package.json#L206) file.

There's no workaround yet. What is the expected display name, "Use TS Go"?

Reply:
Worth noting users don’t need to see this setting if they use this
<img width="629" alt="image" src="https://github.com/user-attachments/assets/6a6e2c09-e30f-4761-8b88-7e11beda9ef0" />